sdk/healthdataaiservices/azure-health-deidentification/README.md

```diff
@@ -69,40 +69,39 @@ DeidentificationClient deidentificationClient = deidentificationClientbuilder.bu
 
 ### Calling `Deidentification` endpoint
 
-Calling the realtime endpoint with an input.
+Calling the realtime endpoint with some input text.
 
 ```java com.azure.health.deidentification.sync.helloworld
 String inputText = "Hello, my name is John Smith.";
-
 DeidentificationContent content = new DeidentificationContent(inputText);
 
-DeidentificationResult result = deidentificationClient.deidentify(content);
+DeidentificationResult result = deidentificationClient.deidentifyText(content);
```

Comment:
The change from `deidentify` to `deidentifyText` looks like a breaking change, should we mention it in CHANGELOG?